src/Template/Loader/FractalCompoundHandlesLoader.php

```diff
@@ -129,9 +141,17 @@ private function convertToTwigPath($handle) {
 
     $path = [
       $activeTheme->getPath(),
-      reset($libs[$namespace]['paths']) . $componentName,
-      $filename . self::TWIG_EXTENSION
+      reset($libs[$namespace]['paths']),
     ];
+    if (strpos($filename, '--') === FALSE) {
+      $path[] = $componentName;
+    }
+    else {
+      $path = array_merge($path, $subpaths);
+      $variantParts = explode(self::VARIANT_DELIMITER, $filename);
+      $path[] = $variantParts[0];      
+    }
+    $path[] = $filename . self::TWIG_EXTENSION;
```

Comment:
Enter new line and close to next `$path` expression
```suggestion

    $path[] = $filename . self::TWIG_EXTENSION;
```